server/controllers/inventory/import/index.js

```diff
@@ -35,15 +34,15 @@ function downloadTemplate(req, res, next) {
  */
 function importInventories(req, res, next) {
   if (!req.files || req.files.length === 0) {
-    next(new BadRequest('Expected at least one file upload but did not receive any files.'));
+    next(new BadRequest('Something broke', 'ERRORS.EVERYTHING_BAD'));
```

Comment:
You took what I said a bit too literally ;).  The 'Expected at least one file upload but did not receive any files.' is a perfect translation, but we need to send a key with that translation back to the client.  So, something like:
```js
next(new BadRequest('Expected at least one file upload but did not receive any files.', 'ERRORS.MISSING_UPLOAD_FILES'));
```
where `ERRORS.MISSING_UPLOAD_FILES` is "Expected at least one file upload but did not receive any files." in English.  Does that makes sense?

Reply:
I think you forgot to correct this!